pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_installing_uploader_tool.adoc

```diff
@@ -0,0 +1,34 @@
+[id="installing-uploader-tool_{context}"]
+
+= Installing the Uploader tool
+
+.Procedure
+
+. Download the Uploader script.
++
+[options="nowrap" subs="normal"]
+----
+$ curl -S https://raw.githubusercontent.com/redhataccess/pantheon/master/uploader/setup.sh
```

Comment:
Wouldn't wget be better than curl? curl only displays the file content and there is then an additional step to store the displayed contents in a file.

If using curl, I suggest modifying the step to:

curl -S https://raw.githubusercontent.com/redhataccess/pantheon/master/uploader/setup.sh > setup.sh

so that we don't need another step to store the script.

Reply:
@benradey I refer @theashiot's question about curl vs wget to you. For now, I'm taking the suggestion and changing curl to wget.

Reply:
The intention is to pipe the script to bash, not redirect to a file. But it works either way. I will admit that piping to bash is generally bad practice, but I would consider us a trustworthy source, IMHO.